src/assets/style.scss

```diff
@@ -181,9 +181,18 @@ pre {
 
 table {
   border-collapse: collapse;
+  margin-bottom: $padding-containers;
```

Comment:
It's typical to give flow-like elements (e.g. those which are part of a body of text, like figures, paragraphs, headings, lists) a bottom margin. I'm giving the table element some margin so that any text (or end of article) which follows it is not crammed right against the bottom of the table.